Added Basic Setup of the Sensor Configuration

msk610 opened the pull request:
I added a new fragment class called ```ConfigFragment```. This fragment is going to deal with updating the sensor settings. I utilized the ```SensorService``` class to get the list of sensors. I parsed the JSON array to get the sensor name and added it to a ```scroll view``` inside of the fragment. I added to a scroll view so you can scroll down the list of sensors. Each sensor has a seek bar/slider along with it. For now, when the app is opened, the slider is set to the power value of the sensor and the listener for the slider does not do anything. After determining how to update the sensor settings using the slider, that functionality will be added and pushed to the config-ui feature branch. Screenshot:
![image](https://cloud.githubusercontent.com/assets/10778552/24739419/eb97db48-1a69-11e7-96ec-523773a40773.png)

msk610 committed:
Setup basic configuration layout

lukpueh commented:
This is great!

Let's keep in mind that we will need a mapping of generic sensor types known to the clearinghouse  and the actual sensors available on the device, i.e.:

if an experimenter requests devices with `accelerometer`, the clearinghouse should  serve e.g.:  [an LG H420  with an `ACCELEROMETER`](https://github.com/aaaaalbert/sensibility-testbed/wiki/lg-h420--5.0.1), or a [Samsung S4 with a `K330 3-axis Accelerometer`](https://github.com/aaaaalbert/sensibility-testbed/wiki/samsung-s4--5.0.1) or a [Nexus 6 with an `Invensense Accelerometer`](https://github.com/aaaaalbert/sensibility-testbed/wiki/nexus-6--7.1.1), ...

msk610 commented:
I can get the sensor type from the json I parsed from SensorService. So, I can do that fairly easily. I think the confusing part for me is just determining what to do with the slider when a user moves it and what should it represent.

msk610 committed:
Parse and display networks using misc service along with sliders

msk610 commented:
Added layout updates such as two sliders for each sensor.  Utilized ```MiscService``` to grab networks data such as WIFI and Bluetooth. Separated everything into two categories, for now, networks and sensors. Also added low, medium, high buttons that can be used to automatically reset the sensor and network settings. No functionality added for those buttons or the sliders for now. Screenshot: 
![image](https://cloud.githubusercontent.com/assets/10778552/25467423/15f8c938-2adc-11e7-9f09-5855e52413d9.png)

msk610 committed:
Add naming change for UI
